examples/numgen.py

```diff
@@ -86,8 +87,14 @@ def __init__(self, name: str, table: str = "number", parent: Optional[QObject] =
         self.generatorFrame.generateButton.clicked.connect(self.generateNumber)
 
     def frames(self) -> Tuple[GeneratorFrame, ViewerFrame]:
```

Comment:
Now this type hinting becomes incorrect.
The correct type would be:
```python
Union[Tuple[GeneratorFrame, ViewerFrame], Tuple[GeneratorFrame]]
```

Reply:
I applied it, again!